docs/azure/deployment.md

```diff
@@ -15,7 +15,7 @@ Whether your workflow is through the [Azure CLI](https://learn.microsoft.com/cli
 
 ## Deployment tutorials
 
-The tutorials below, hosted on the Microsoft Docs platform, walk you through different ways of creating and deploying apps to Azure using Visual Studio Code:
+These tutorials from Microsoft Learn show different ways of creating and deploying apps to Azure via Visual Studio Code:
```

Comment:
Suggest not using "show" in order to be more inclusive to non-sighted readers.
If you think "walk you through" is too conversational, perhaps "teach you" or "describe".

Reply:
Fixed